tests/CSharpTemp/CSharpTemp.h

```diff
@@ -700,3 +700,10 @@ class DLL_API TestVariableWithFixedArrayType
 public:
 	static Foo variableWithFixedArrayType[2];
 };
+
+class DLL_API TestOutTypeInterfaces
```

Comment:
It alone doesn't create the interfaces. I am using ```TestParamToInterfacePassBaseTwo``` which  is secondary base for ```TestParamToInterfacePass``` class and hence is marshalled as interface by paramToInterfacePass...
I thought it would be better to use existing classes instead of creating dummy classes.